pep/types/@gavant/ember-button-spinner/component.d.ts

```diff
@@ -0,0 +1,12 @@
+import Component from '@glint/environment-ember-loose/glimmer-component';
+
+import { BaseGlimmerSignature } from 'pep/utils/types';
+
+import type { ButtonSpinnerArgs } from '@gavant/ember-button-spinner/components/button-spinner';
+export declare class ButtonSpinnerComponent extends Component<BaseGlimmerSignature<ButtonSpinnerArgs>> {}
+
+declare module '@glint/environment-ember-loose/registry' {
+    export default interface Registry {
+        ButtonSpinner: typeof ButtonSpinnerComponent;
+    }
+}
```

Comment:
we should probably port these over to the actual addons if possible, when we get a chance.

Reply:
I have mixed feelings about that, but we can discuss in chat. Resolving for now though.